.github/workflows/post-commit.yml

```diff
@@ -0,0 +1,94 @@
+# A workflow to run against renovate-bot's PRs,
+# such as `make package` after it updates the package.json and package-lock.json files.
+# The potentially untrusted code is first run inside a low-privilege Job, and the diff is uploaded as an artifact.
+# Then a higher-privilege Job applies the diff and pushes the changes to the PR.
+# It's important to only run this workflow against PRs from trusted sources, after also reviewing the changes!
+
+name: Post-Commit
+
+on:
+    workflow_dispatch:
+        inputs:
+            pr_number:
+                description: "The pull request number."
+                required: true
+                type: number
+
+env:
+    COMMAND: |
+        (
+            cd ./actions/installer/dist/../ && \
+            make clean && \
+            make package && \
+            true
+        )
+    COMMIT_MESSAGE: "apply post-commit changes"
+    ARTIFACT: changes.patch
+
+jobs:
+    diff:
+        permissions:
+            pull-requests: read
+        outputs:
+            patch_not_empty: ${{ steps.diff.outputs.patch_not_empty }}
+        runs-on: ubuntu-latest
+        steps:
+            - name: checkout
+              uses: actions/checkout@v4
+              with:
+                  repository: ${{ github.repository }}
+                  persist-credentials: false
+            - name: checkout-pr
+              env:
+                  GH_TOKEN: ${{ github.token }}
+              run: |
+                  gh pr checkout ${{ inputs.pr_number }}
+            - name: run-command
+              run: ${{ env.COMMAND }}
```

Comment:
I think it's fine to copy the command instead of using env.COMMAND. Or is there a reason for using an env variable here but not in the diff step?

Reply:
No real reason other than it looks nicer to have the COMMAND as a veriable at the top of this document.

Reply:
changed.